src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -56,6 +56,7 @@ To install our Kubernetes integration, we recommend that you follow the instruct
     <TabsBarItem id="2">In Windows</TabsBarItem>
```

Comment:
Leaving a call out about the right nav defaulting to the h2s in the guided install steps, even if you click across different install options. Until Eng has capacity to make the right nav dynamic with the horiz tabs, we might consider switching the headers in step component to h3s.